packages/core/src/lists.ts

```diff
@@ -0,0 +1,121 @@
+import { css } from '@emotion/react';
+import colors from './colors';
+import fonts from './fonts';
+
+const generateListResets = () => {
+  let styles = '';
+  for (let $i = 0; $i < 50; $i++) {
+    styles += ` 
+      ol.ol-reset-${$i} { counter-reset: item ${$i - 1} }  
+    `;
+  }
+
+  return styles;
+};
+
+const listsStyle = css`
+  // Parent shared styles/defaults
+  ul,
+  ol {
+    margin-top: 0;
+    margin-left: 0;
+    ${fonts.sizes('18px', '29px')};
+
+    // Due to displayment issues in the editor. The editor wraps all text with a <p> tag
+    p {
+      display: inline;
+    }
+    // Child unordered lists
+    ul {
+      padding-left: 20px;
+    }
+
+    // Child ordered lists
+    ol {
+      padding-left: 20px;
+    }
+  }
+
+  // List item
+  li {
+    margin-top: 24px;
+  }
```

Comment:
Kunne evetuelt ha det som er felles her og fjerne fra &:before dei to plassene under
```suggestion
  li {
    margin-top: 24px;
    &:before {
      position: absolute;
      transform: translateX(-100%);
      padding-right: 0.25em;
    }
  }
```